invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/index.js

```diff
@@ -12,3 +12,9 @@ export { Results } from "./Results";
 export { SearchBar } from "./SearchBar";
 export { SearchApp, SearchAppFacets, SearchAppResultsPane } from "./SearchApp";
 export { SearchConfigurationContext } from "./context";
+export {
+  MultipleOptionSearchBar,
```

Comment:
I would change `Option` to `Options` (plural) everywhere.
```suggestion
  MultipleOptionsSearchBar,
```